services/grpc/lightning.subscriptions.js

```diff
@@ -83,6 +84,36 @@ function subscribeTransactions() {
   })
   return call
 }
+/**
```

Comment:
insert newline above

Reply:
done

Reply:
Still need newline above start of comment